### ISSUE TITLE
Input /examples/rigid/in.rigid.tnr crashes with Kokkos CUDA

### ISSUE THREAD

**Issue by stanmoore1** [lammps/lammps#357]
/examples/rigid/in.rigid.tnr runs fine with Kokkos OpenMP but crashes on the GPU on around timestep 7100: 
<pre>
    7100     6.834257    6.0974715    11.465414  0.061240709    13.494205    57.033279    57.033279    57.033279  0.056499479  0.075608169   0.05161448
WARNING: FENE bond too long (../bond_fene_kokkos.cpp:136)
</pre>
Then the thermo output goes NaN and the NaNs spread…

**Comment by ibaned** [comment]
running with the changes in #363 does not exhibit the bond warning or any NaNs, so I would say in that sense this issue will be resolved once #363 is merged. However, the console output is still slightly different between the CUDA+Kokkos and serial versions. That can probably be considered a separate issue.

**Comment by stanmoore1** [comment]
OK, agreed.

**Comment by stanmoore1** [comment]
Fixed via PR #363. Closed #357.
